src/Nethermind/Nethermind.Api/IApiWithBlockchain.cs

```diff
@@ -48,7 +48,7 @@ public interface IApiWithBlockchain : IApiWithStores, IBlockchainBridgeFactory
         IFilterStore? FilterStore { get; set; }
         IFilterManager? FilterManager { get; set; }
         IHeaderValidator? HeaderValidator { get; set; }
-        ITrieStore? ReadOnlyTrieStore { get; set; }
+        ReadOnlyTrieStore? ReadOnlyTrieStore { get; set; }
```

Comment:
not a great fan of non-interface in API

Reply:
yes and no, I see this as a special case as this is wrapper just for ReadOnlyness